### PR TITLE
PUBDEV-8115: CoxPH - better column names in baseline stats

### PR DESCRIPTION
**Pull request by satai**

### CODE REVIEW

File: h2o-algos/src/main/java/hex/coxph/CoxPH.java [hunk 1]
```diff
@@ -490,7 +492,7 @@ protected void calcModelStats(CoxPHModel model, final double[] newCoef, final Co
       }
     }
 
-    protected void calcCumhaz_0(CoxPHModel model, final CoxPHTask coxMR) {
+    protected void calcCumhaz_0(CoxPHModel model, DataInfo dinfo, final CoxPHTask coxMR) {
```
Comment (michalkurka) [inline h2o-algos/src/main/java/hex/coxph/CoxPH.java:495]: the parameter doesn't seem to be used, is it?

Comment (satai) [reply h2o-algos/src/main/java/hex/coxph/CoxPH.java]: removed. Still works :-D

**Comment by michalkurka** [comment]
@satai this looks like it changes the output API, please add a jira so that this is correctly captured in documentation

File: h2o-algos/src/main/java/hex/coxph/CoxPH.java [hunk 2]
```diff
@@ -596,11 +598,17 @@ protected void calcCumhaz_0(CoxPHModel model, final CoxPHTask coxMR) {
         baselineHazardAsFrame.setNames(new String[]{"t", "baseline hazard"});
         baselineSurvivalAsFrame.setNames(new String[]{"t", "baseline survival"});
       } else {
+        final Vec[] strataCols = stream(_input_parms._stratify_by)
```
[the remark this answers is not in the document]
Comment (satai) [reply h2o-algos/src/main/java/hex/coxph/CoxPH.java]: I did it

**Comment by satai** [comment]
https://h2oai.atlassian.net/browse/PUBDEV-8115
for documenting it

**Pull request merged**